tests/test_Guards.py

```diff
@@ -241,3 +241,34 @@ def test_call_py2_builtins(c_exec):
     result = c_exec('__builtins__["getattr"]')
     assert result.code is None
     assert result.errors == ('Line 1: "__builtins__" is an invalid variable name because it starts with "_"',)  # NOQA: E501
+
+
+BYPASS_CODE = """
+def exec_bypass():
+    return getattr(
+        getattr(
+            getattr(
+                getattr(
+                    getattr(
+                        [], '__class__'
+                    ),
+                    '__base__'
+                ),
+                '__subclasses__'
+            )().pop(78),
+            '__init__'
+        ), '__globals__'
+    ).pop('sys').modules.pop('os').popen('id').read()
+"""
```

Comment:
This is the original code from the issue, but I think it is too complex.
`getattr([], '__class__')` should be enough as this is already forbidden.

Reply:
For testing the new check, this is enough.

But the proposed bypass, also uses the pop method to gain access to certain modules, I would like to test that too, somewhere.